EIPS/eip-4762.md

```diff
@@ -236,11 +236,17 @@ When executing a transaction, maintain four sets:
 
 When an **access** event of `(address, sub_key, leaf_key)` occurs, perform the following checks:
 
+ * Perform the following steps unless `address` is either:
+   * the transaction's originator;
+   * the transactions's target;
+   * the address of the originator of a `CALL`, `STATICCALL`, `DELEGATECALL` and `CALLCODE` instruction; or,
+   * the address of the target of a `CALL`, `STATICCALL`, `DELEGATECALL` and `CALLCODE` instruction.
```

Comment:
```suggestion
```

Reply:
so any access related to originator and target won't show up? how will stateless clients run?

Reply:
they are added to the witness, but no access cost is charged for them